includes/restore.js

```diff
@@ -48,14 +49,29 @@ module.exports = function(dbClient, options, readstream, ee) {
     }
   });
 
-  return pipeline(
+  const batchPreparationStreams = [
     readstream, // the backup file
     new Liner(), // line by line
     new MappingStream(restore.backupLineToDocsArray), // convert line to a docs array
     new BatchingStream(options.bufferSize, true), // make new arrays of the correct buffer size
-    new MappingStream(restore.docsToRestoreBatch), // make a restore batch
+    new MappingStream(restore.docsToRestoreBatch) // make a restore batch
+  ];
+  const mappingStreams = [];
+  const restoreStreams = [
     new MappingStream(restore.pendingToRestored, options.parallelism), // do the restore at the desired level of concurrency
     output // emit restored events
+  ];
+
+  if (options.attachments) {
+    mappingStreams.push(
+      new MappingStream(new Attachments().encode, options.parallelism)
```

Comment:
I can't because it's suppsoed to be `decode` 🙈 - although it seems the core can handle either type.
Fixed in b104b5e9d1574739d82d912a8bc03f0b7b951bfc

Reply:
Thanks! I was unsure whether it was intentional, or just a typo here. Better safe than sorry. :)